andela/wg-baqi
# #166151265 Muscle List Could be Updated

#### Description
Update muscle list with an additional list of muscles
#### Type of change
- [x] Currently, there are only 16 muscles in the muscle list. These can be viewed from the muscle .svg image on the muscle overview page or via a rest client.
- [x] This feature will add three more muscles to this list
#### How Has This Been Tested?
- Clone [this repo](https://github.com/andela/wg-baqi.git) to your local computer
- CD into `wger` and create a virtual env by doing `python3 -m virtualenv venv`
- Activate the virtual env: `source venv/bin/activate` && install requirements `pip install -r requirements.txt`
- Run migrations by invoking the `bootstrap-wger` command as is on the Readme.rst
- In your browser, navigate to `http://127.0.0.1:8000/en/exercise/muscle/overview/`
- Compare the diagram with the [original wger website](https://wger.de/en/exercise/muscle/overview/)
- You should see more muscles
#### Any background context:
None
#### Story ID
[#166151265](https://www.pivotaltracker.com/story/show/166151265)
#### Screenshots
<img width="916" alt="Screen Shot 2019-06-06 at 21 29 43" src="https://user-images.githubusercontent.com/38868860/59057198-89d49280-88a2-11e9-90b3-f99ebc60340e.png">
<img width="939" alt="Screen Shot 2019-06-06 at 21 29 52" src="https://user-images.githubusercontent.com/38868860/59057199-89d49280-88a2-11e9-94a6-2ec3a495f23f.png">
